source/system-administrators/activities/reindexing-es/reindex-3-1-10-to-3-1-17.rst

```diff
@@ -0,0 +1,125 @@
+:is-up-to-date: True
+
+:orphan:
+
+.. document does not appear in any toctree, this file is referenced
+   use :orphan: File-wide metadata option to get rid of WARNING: document isn't included in any toctree for now
+
+.. _reindex-content-for-3-1-10-to-3-1-17:
+
+--------------------------------------------------------------------------------
+Reindexing Content Using Elasticsearch for Crafter CMS versions 3.1.10 to 3.1.17
+--------------------------------------------------------------------------------
+
+The steps listed below for reindexing content without disrupting service in production using Elasticsearch applies only to Crafter CMS versions 3.1.10 to 3.1.17
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Step 1: Create a new index
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to create a new index on Elasticsearch where you can index the content:
+
+#. Check the next version for the index name, you can find the current name using the following command:
+
+   .. code-block:: bash
+      :linenos:
+
+          curl -s http://ES_HOST:ES_PORT/_cat/aliases | grep SITE_NAME
+
+      # Example result
+
+      SITE_NAME   SITE_NAME_v1   - - -
+
+#. Download the Elasticsearch mappings appropriate for the index. If the index is an authoring index (it has an
+   -authoring suffix), then use ``authoring-mapping.json``. If it's any other index (it has a -preview suffix
+   or no suffix), then use ``default-mapping.json``. The latest version of the mappings can be found
+   `here <https://github.com/craftercms/search/tree/v3.1.17/crafter-search-elasticsearch/src/main/resources/crafter/elasticsearch>`_
+
+#. Create the new index using the next version and the settings file:
+
+   .. code-block:: bash
+      :linenos:
+
+      curl -s -X PUT http://ES_HOST:ES_PORT/SITE_NAME_v2 -H 'Content-Type: application/json' -d '@default-index-settings.json'
```

Comment:
This example is not accurate, because the JSON file only contains the mappings the content needs to be copied to the actual request body instead of referencing the file